app/services/elastic_search_finder.rb

```diff
@@ -2,6 +2,10 @@
 
 class ElasticSearchFinder
   def call(query, sort)
-    Vacancy.__elasticsearch__.search(size: 1000, query: query, sort: sort)
+    Vacancy.__elasticsearch__.search(size: size, query: query, sort: sort)
+  end
+
+  def size
+    Vacancy.default_per_page
```

Comment:
Tap into the Kaminari configuration set on the Vacancy model with `paginates_per 10` to make it update automatically without being forgotten.